packages/framework/src/Resources/views/Admin/Form/productParameterValue.html.twig

```diff
@@ -1,53 +1,51 @@
                         {{ localeFlag(locale) }}
                     </span>
-                    {{ form_errors(valueTextLocaleForm, { errors_attr: { class: 'table-form__cell__error' } } ) }}
+                    {{ form_errors(valueTextLocaleForm, { errors_attr: { class: 'table-params__cell__error' } } ) }}
                 </div>
             {% endfor %}
-        </td>
-        <td class="table-form__cell table-form__cell--actions">
-            <a href="#" class="js-parameters-item-remove table-form__action"><i class="svg svg-delete-thin in-icon"></i></a>
-        </td>
-    </tr>
+        </div>
+        <div class="table-params__cell table-params__cell--actions">
+            <a href="#" class="js-parameters-item-remove table-params__action"><i class="svg svg-delete-thin in-icon"></i></a>
+        </div>
+    </div>
 {% endmacro %}
 
 {% block product_parameter_value_widget %}
     {% import _self as self %}
     <div class="js-form-group form-group">
-        <table class="table-form" id="product_form_parameters">
-            <thead>
-            <tr class="table-form__row">
-                <th class="table-form__cell table-form__cell--del-before table-form__cell--label">{{ 'Parameter'|trans }}</th>
-                <th class="table-form__cell table-form__cell--del"></th>
-                <th class="table-form__cell table-form__cell--del-after">{{ 'Value'|trans }}</th>
-                <th class="table-form__cell"></th>
-            </tr>
-            </thead>
-            <tbody class="js-parameters" data-prototype="{{ self.parameterRow(form.vars.prototype)|escape }}" data-index="{{ form|length }}">
+        <div class="table-params" id="product_form_parameters">
+            <div class="table-params__row">
+                <th class="table-params__cell table-params__cell--header"><span class="table-params__cell__label">{{ 'Parameter'|trans }}</span></th>
+                <th class="table-params__cell table-params__cell--header"></th>
+                <th class="table-params__cell table-params__cell--header"><span class="table-params__cell__label">{{ 'Value'|trans }}</span></th>
+                <th class="table-params__cell table-params__cell--header"></th>
+            </div>
+            <div class="js-parameters" data-prototype="{{ self.parameterRow(form.vars.prototype)|escape }}" data-index="{{ form|length }}">
 
             {% for key, parameter in form %}
                 {{ self.parameterRow(parameter, key) }}
             {% else %}
                 {% do form.setRendered %}
             {% endfor %}
 
-            <tr class="js-parameters-empty-item table-form__row">
-                <td class="table-form__cell" colspan="4">{{ 'Add some parameters'|trans }}</td>
-            </tr>
-            </tbody>
-        </table>
+                <div class="js-parameters-empty-item table-params__row">
+                    <div class="table-params__cell" colspan="4">{{ 'Add some parameters'|trans }}</div>
```

Comment:
Atribute `colspan` in div element?

Reply:
Probably it was table in past. Changed. Thx